### DISCUSSION TITLE
Rejoin uBO organisation

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#3093] [General] [answered]
@gwarser @okiehsch Hey, can you one of you add me back ? GitHub removed me because I didn't have 2FA enabled after the organisation enabled it and had a fixed deadline.

**Comment by okiehsch** [comment, marked as answer]
Done.

**Reply by uBlock-user** [reply]
Thanks.

**Reply by uBlock-user** [reply]
One question -  Is it specific to a repository in the org ?

**Reply by okiehsch** [reply]
You should have read access to all repos and write access to `uBlock-issues`.

**Reply by uBlock-user** [reply]
> write access to uBlock-issues.

Yes, it seems, I only have write access to that repository only currently, while previously it was to all other repositories too.

**Reply by okiehsch** [reply]
If I invite someone I have to choose teams.
I thought you don't want to maintain the filter lists anymore, that's why I added you to  [uBlock-issues volunteers](https://github.com/orgs/uBlockOrigin/teams/ublock-issues-volunteers).
You have to be member of [uBlock-filters volunteers](https://github.com/orgs/uBlockOrigin/teams/ublock-filters-volunteers) to get write access to `uAssets`.

**Reply by uBlock-user** [reply]
> I thought you don't want to maintain the filter lists anymore

Yes, due to my disagreements with mapx

> You have to be member of [uBlock-filters volunteers](https://github.com/orgs/uBlockOrigin/teams/ublock-filters-volunteers) to get write access to uAssets.

Okay, I see.

**Comment by uBlock-user** [comment]
@okiehsch Have you left the uBlockOrigin organisation ? I don't see you in the group at all.

**Reply by okiehsch** [reply]
Yes. I told gorhill and gwarser just before I removed myself from uBlockOrigin.

**Reply by okiehsch** [reply]
> he's out from reddit and slack too

I wasn't active on reddit for some years anyway and I see no point staying on slack if I stop maintaining a filter list.

**Reply by Yuki2718** [reply]
Thank you for all your contribution from the dawn of this project.

**Reply by okiehsch** [reply]
Thx. I appreciate that.

**Reply by uBlock-user** [reply]
> Yes

May I know why, if you don't mind me asking.

**Reply by okiehsch** [reply]
It wasn't one thing.
I just lost the motivation to continue maintaining over the last few months.
I started actively maintaining uAssets over seven years ago, it's just time to stop for me.

**Reply by uBlock-user** [reply]
Hmmm, yeah I remember seeing you join the org sometime in 2016-17, so it's been a long time. Anyway, good luck in your future endeavours.

**Reply by okiehsch** [reply]
Thx.

**Reply by stephenhawk8054** [reply]
Thank you for all your help to the project :)

**Reply by iam-py-test** [reply]
Thank you for making the internet a better place.

**Reply by gorhill** [reply]
> I started actively maintaining uAssets over seven years ago

I remember very well, you were [the first I gave commit rights to the repo](https://github.com/uBlockOrigin/uAssets/issues/259#issuecomment-271122978)<sup>[1]</sup>, at a time I was discouraged that there was all these abilities added to uBO but no filter list maintainers to use them. Not sure I would have had the motivation to keep going back then without your timely help given [the burden I was feeling at the time](https://github.com/uBlockOrigin/uAssets/graphs/contributors?from=2014-06-22&to=2017-01-31&type=c).

<sub>[1] I have this link bookmarked since a long time because I consider it was a key positive milestone in the project</sub>

**Reply by MasterKia** [reply]
Thank you, you've been a good mentor along the way.
